src/main/parser.js

```diff
@@ -1,38 +1,38 @@
 "use strict";
 
 const path = require("path");
-const ConfigError = require("./errors").ConfigError;
+const ConfigError = require("../common/errors").ConfigError;
 
 const parsers = {
   get flow() {
-    return eval("require")("./parser-flow");
+    return eval("require")("../language-js/parser-flow");
   },
   get graphql() {
-    return eval("require")("./parser-graphql");
+    return eval("require")("../language-graphql/parser-graphql");
   },
   get parse5() {
-    return eval("require")("./parser-parse5");
+    return eval("require")("../language-html/parser-parse5");
   },
   get babylon() {
-    return eval("require")("./parser-babylon");
+    return eval("require")("../language-js/parser-babylon");
   },
   get typescript() {
-    return eval("require")("./parser-typescript");
+    return eval("require")("../language-js/parser-typescript");
   },
   get css() {
-    return eval("require")("./parser-postcss");
+    return eval("require")("../language-css/parser-postcss");
   },
   get less() {
-    return eval("require")("./parser-postcss");
+    return eval("require")("../language-css/parser-postcss");
   },
   get scss() {
-    return eval("require")("./parser-postcss");
+    return eval("require")("../language-css/parser-postcss");
   },
   get json() {
-    return eval("require")("./parser-babylon");
+    return eval("require")("../language-js/parser-babylon");
   },
   get markdown() {
-    return eval("require")("./parser-markdown");
+    return eval("require")("../language-markdown/parser-markdown");
```

Comment:
In bundled version: `Cannot find module '../language-markdown/parser-markdown' from 'index.js'`

Reply:
Yeah I'll be moving the output too eventually. Will have to upgrade the playground too.